### DISCUSSION TITLE
Which --log-level option is most verbose?

### DISCUSSION THREAD

**Discussion by Mr-Ples** [appium/appium#17571] [Q&A]
Trying to debug some errors and wondering which Appium2 server CLI log level (see options: https://appium.github.io/appium/docs/en/2.0/cli/args/) is the most detailed and preferably includes the other levels. Is it possible to have multiple log levels active at once?

**Comment by jlipps** [comment]
As the new docs indicate (https://appium.github.io/appium/docs/en/2.0/cli/args/), the server default is already the most verbose level (`debug`). You can have multiple log levels active; one for console and one for file.
